### ISSUE TITLE
bug (reddit): Failed to apply sanitize-sharing-links

### ISSUE THREAD

**Issue by TheWitch69** [revanced/revanced-patches#2568]
### Type

Error while patching

### Bug description

Failed to apply sanitize-sharing-links

### Steps to reproduce

Use manager 1.3.8

Use Reddit 2023.25.1 & 2023.25.0 with the default patches and fail. 2023.24.0 work.

### Relevant log output

```shell
Initializing installer
Creating working directory
Copying original apk
Unpacking input apk
Reading dex files
Decoding AndroidManifest.xml only, because resources are not needed
Merging integrations
Deleting existing resource cache directory
Decoding resources
Executing patches
Applied disable-screenshot-popup
Applied enable-android-debugging
Applied export-all-activities
Applied hide-ads
Applied predictive-back-gesture
Applied premium-icon-reddit
Applied remove-screen-capture-restriction
Applied remove-screenshot-restriction
Failed to apply sanitize-sharing-links: Failed to resolve ShareLinkFactoryFingerprint
Repacking patched apk
Compiling resources
Writing modified dex files
Finished!
```

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by LunaSquee** [comment]
Longer stack trace from revanced builder:
```
app.revanced.extensions.ExtensionsKt.toErrorResult(Extensions.kt:22) at 
app.revanced.patches.reddit.misc.tracking.url.patch.SanitizeUrlQueryPatch.execute(SanitizeUrlQueryPatch.kt:42) at 
app.revanced.patches.reddit.misc.tracking.url.patch.SanitizeUrlQueryPatch.execute(SanitizeUrlQueryPatch.kt:19) at 
app.revanced.patcher.Patcher.executePatches$executePatch(Patcher.kt:333) at 
app.revanced.patcher.Patcher.access$executePatches$executePatch(Patcher.kt:38) at 
app.revanced.patcher.Patcher$executePatches$1.invokeSuspend(Patcher.kt:358) at 
kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) at 
kotlin.sequences.SequenceBuilderIterator.hasNext(SequenceBuilder.kt:129) at 
app.revanced.utils.patcher.PatcherKt.applyPatchesVerbose(Patcher.kt:84) at 
app.revanced.cli.patcher.Patcher.start$revanced_cli(Patcher.kt:19) at 
app.revanced.cli.command.MainCommand.run(MainCommand.kt:166) at 
picocli.CommandLine.executeUserObject(CommandLine.java:2026) at 
picocli.CommandLine.access$1500(CommandLine.java:148) at 
picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461) at 
picocli.CommandLine$RunLast.handle(CommandLine.java:2453) at 
picocli.CommandLine$RunLast.handle(CommandLine.java:2415) at 
picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273) at 
picocli.CommandLine$RunLast.execute(CommandLine.java:2417) at 
picocli.CommandLine.execute(CommandLine.java:2170) at app.revanced.cli.main.MainKt.main(Main.kt:7)
```

**Comment by KobeW50** [comment]
Until the fix is merged to main, you can use version 2023.21.0. I tested it and it works.

<https://www.apkmirror.com/apk/redditinc/reddit/reddit-2023-21-0-release/reddit-2023-21-0-2-android-apk-download/download/>

[revanced-manager_patcher_20230703130645671882.txt](https://github.com/revanced/revanced-patches/files/11939741/revanced-manager_patcher_20230703130645671882.txt)
